tests/acceptance/features/apiOcm/acceptInvitation.feature

```diff
@@ -135,3 +93,56 @@ Feature: accepting invitation
         }
       }
       """
+
+  @env-config
+  Scenario: user cannot accept expired invitation tokens
+    Given using server "LOCAL"
+    And the config "OCM_OCM_INVITE_MANAGER_TOKEN_EXPIRATION" has been set to "1s"
+    And "Alice" has created the federation share invitation
+    When using server "REMOTE"
+    And wait "2" seconds for the token to expire
```

Comment:
I decided to make this a separate explicit step and set a timeout here instead of embedded it to createInvite or acceptInvite methods.